nextflow_schema.json

```
@@ -233,6 +229,10 @@
         "igenomes_ignore": {
             "type": "boolean",
             "default": false
+        },
+        "round_DE": {
+            "type": "string",
```

Type `string` might lead to confusions. You could add a help text like this:
```suggestion
            "help": "integer as string",
```

---

I honestly don't remember myself why this is a string...changing it to int.